mission-statement.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: pages
+---
+
+Mission Statement
+=================
+
+Our mission at Fullstack LA is simple.
+
+> **<em>We want to help developers with their first, tenth, and hundredth contribution.</em>** 
+
+As a community, we value skill building, inclusivity, and a commitment to open source projects that not only teach you something new, but help get your projects shipped asap.
+
+We truly believe that everyone has something to contribute -- pun intended 😏. Whether you're a seasoned open source committer/maintainer, we want you to come show us how it's done. 
+
+Or if you’ve never pushed code to an open source project, we still want you to come. We’re here to serve you, the coding community, by helping you find projects, identify issues, and understand the process of contributing to open source projects.
+
+We are Fullstack LA, and we want YOU to join us!😎 
```

Comment:
This is great!
I've been noodling with this as well. I think the line "We want to help developers with their first, tenth, and hundredth contribution." is specific to the Open source event and we should use it in that description. 
Where I'm thinking at the moment is core mission of Fullstack is more centered around mentorship, skill building and community. There is more to talk about here... I'm still thinking.

I'm going to add this page without a link to it so we have a place to continue the conversation.